install.sh

```diff
@@ -9,7 +9,7 @@
 # It runs on Unix shells like {a,ba,da,k,z}sh. It uses the common `local` extension.
 # Note: Most shells limit `local` to 1 var per line, contra bash.
 #
-# Most of this code is based on/copy-pasted from rustup and protostar installers.
+# Most of this code is based on/copy-pasted from rustup installer.
```

Comment:
This notice actually stays true :D We still have stuff copied from OG Protostar installer